js-miniapp-sdk/README.md

```diff
@@ -411,6 +411,60 @@ miniApp.chatService.sendMessageToMultipleContacts(messageToContact)
 
 ## Advanced Usage
 
+### Errors management
+
+#### Access Token error
+
+Error messages sent to the bridge when a `getAccessToken` call is failing should always be in the following JSON format:
+
+````json
+{
+  "message": "error_message",
+  "type": "error_type_key"
+}
+````
+At the moment, the SDK will throw a `MiniAppError` (and subclasses), subclassing `Error` type, only on `getAccessToken` failures, based on `type` value.
+If the error `type` value is not supported, `Error.name` field will automatically be set to `Other`.
```

Comment:
I think this seems like internal implementation details for the SDK - so maybe it's not needed here.

Reply:
removed